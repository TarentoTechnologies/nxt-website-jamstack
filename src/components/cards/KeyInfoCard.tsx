import { GatsbyImage, getImage } from "gatsby-plugin-image";
import ReactHtmlParser from "react-html-parser";

import { black75 } from "../../styles/style-guide/ColorGuide.module.css";
import { PrimaryTitle } from "../titles/PrimaryTitle";

interface KeyInfoCardProps {
  data: any;
}

export const KeyInfoCard = ({ data }: KeyInfoCardProps) => {
  return (
    <div className="container-fluid">
      {data.map((listData: any, index: number) => {
        return (
          <div className="px-3 px-sm-3 px-md-0 px-lg-0" key={listData.id}>
            {listData.DescriptionAlignment === "left" ? (
              <div className={`row ${index === 0 ? "" : "my-5"} gx-5`}>
                <div
                  className={`col-sm-12 col-md-6 col-lg-6 ${
                    index === 0 ? "" : "mt-4"
                  }`}
                >
                  <PrimaryTitle
                    title={listData.Title}
                    variant="infoSectionH2"
                  />
                  <div className={`py-2 ${black75}`} data-aos="fade-up">
                    {ReactHtmlParser(
                      listData.Description.data.childMarkdownRemark.html
                    )}
                  </div>
                </div>
                <div
                  className={`col-sm-12 col-md-6 col-lg-6 ${
                    index === 0 ? "" : "mt-4"
                  }`}
                >
                  <GatsbyImage
                    image={
                      getImage(listData.Image.localFile)
                        ? getImage(listData.Image.localFile)
                        : listData.Image.localFile
                    }
                    alt={""}
                    data-aos="fade-up"
                  />
                </div>
              </div>
            ) : (
              <div className={`row ${index === 0 ? "" : "my-5"} gx-5`}>
                <div
                  className={`col-sm-12 col-md-6 col-lg-6 ${
                    index === 0 ? "" : "mt-4"
                  }`}
                >
                  <GatsbyImage
                    image={
                      getImage(listData.Image.localFile)
                        ? getImage(listData.Image.localFile)
                        : listData.Image.localFile
                    }
                    alt={""}
                    data-aos="fade-up"
                  />
                </div>
                <div
                  className={`col-sm-12 col-md-6 col-lg-6 ${
                    index === 0 ? "" : "mt-4"
                  }`}
                >
                  <PrimaryTitle
                    title={listData.Title}
                    variant="infoSectionH2"
                  />
                  <div className={`py-2 ${black75}`} data-aos="fade-up">
                    {ReactHtmlParser(
                      listData.Description.data.childMarkdownRemark.html
                    )}
                  </div>
                </div>
              </div>
            )}
          </div>
        );
      })}
    </div>
  );
};
