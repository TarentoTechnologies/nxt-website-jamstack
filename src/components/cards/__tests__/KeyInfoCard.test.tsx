import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import React from "react";

import { KeyInfoCard } from "../KeyInfoCard";

const cardData = [
  {
    id: "a4302b50-5631-515d-b7fb-9b9bcd55deb8",
    Title: "Challenge",
    Image: {
      localFile: {
        childImageSharp: {
          gatsbyImageData: {
            layout: "constrained",
            placeholder: {
              fallback:
                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAIAAABr+ngCAAAACXBIWXMAAAsTAAALEwEAmpwYAAADnklEQVR42gGTA2z8AL/F0N3g5dXY3ubn69TY3svR2MvQ2OLl6cPI0pSerqCotaKruKu0v7K5xLW7xr3Cy7/Ezb3Dy7q+yLm8xwDZ3eOZo7GnsL+xucWxuMOvt8KKlaXGy9K0vMiTna+ep7W3vsm5wMq5wMnT1t3U1dvM0NfN0djQ1NrV190A0tPXiJSiZ3OBdYGQj5yrwsjQoqq10tfd3+Dkp666k5yrr7K6qK23p665wsfPtLnDsrnEt7zHwMXO09XbANDQ04GJmFlDQlNQV5KZpMTEyZumtMTGzszDwLu4u09QVyspLImPmaSruLzCzIGEjJ6docDCyayyv7q6vwDAxMyCkaS8q621r7ijqLavsLaTnKmSm6mus7zm4+tnYGMAAARKUFmdpLGvtL9aXGW1rK61tbulsL7Av8YA09bdsKKi1MXDwrW4npSXsLS6jpWfgIeSlpmj/f//spiWYTw2lZeeho6aoKGngoWOy8rTmJKYt7zJzra2AN7k7caupdrNyKahpnRobsPIz6mvt5ieqJWXn+Xr+bqYkceilvj8/9XX38HEzKuttZqYnpWTlsGpnbOVhADuztuqi596cW52cniaoKrEw8empKasqrHTurbLp53Xt6j17+zu7/fw8vfe4efi5Ox/f4O/r53TwKbPxLUA1tTbtrjEm5OVlY2OkpWdnp6in56hm4R9oXlsx6iY++TV////9e/t4djc1tvl1NjgfH2DWFRUTkxPmpugAOTo8Obl6tLU2n1/hHBub9DFtu/fyu3dxuPUu8Ovm7CdltHT2sPCyIVzeIlrd8nDznZ5fz1CRlxCT49YaADm6vLq8PbJzNNnanK1q5/o1rzVxKzOu6Sij3x2a2Z3fIe5u8be4u+blqFpVFzIy9WBg4pGRkk4Mzuvm6kA5Onw6OzzzdLYYGNpVFVYd3V3cHBzTmN1RmN/DluTbKfS//z+8PL/y8rVioOKwMbQc3V7Pj9EKi4ytbe6AOLo7+Po7tbb41tfZzc6QHZ3flBpgQBZnQ5ytwBvuS6HxMjT5t7g68bJ1KestLm9xmpobFFRVUFES4iEigDh5u3g5uvb4OhobHQpKzBiYmZMYHYAOW8QLkwKToMAUZMlTnSJnLKtr7imq7StsbqanKObnqePlqKkp64A4+ju4+jt3+Pqf4KISElNY2RpbHF5K1mEM0tlNWOJIXCseoOKsrW7yc3XzM/ZztHaycvVvcHLusDKu8HKcgBLbWfKjJAAAAAASUVORK5CYII=",
            },
            images: {
              fallback: {
                src: "/static/8db79876759e2148da5b7180b91a92d0/1535a/about_copy_bcd2468822.png",
                srcSet:
                  "/static/8db79876759e2148da5b7180b91a92d0/c6d23/about_copy_bcd2468822.png 135w,\n/static/8db79876759e2148da5b7180b91a92d0/ba1a2/about_copy_bcd2468822.png 270w,\n/static/8db79876759e2148da5b7180b91a92d0/1535a/about_copy_bcd2468822.png 540w",
                sizes: "(min-width: 540px) 540px, 100vw",
              },
              sources: [],
            },
            width: 540,
            height: 400,
          },
        },
      },
    },
    DescriptionAlignment: "left",
    Description: {
      data: {
        childMarkdownRemark: {
          html: "<p>Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.</p>\n<p>Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.</p>",
        },
      },
    },
  },
  {
    id: "4ebb7b8a-883c-526a-8ad9-0b1d20fdaef1",
    Title: "Our Approach",
    Image: {
      localFile: {
        childImageSharp: {
          gatsbyImageData: {
            layout: "constrained",
            placeholder: {
              fallback:
                "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAYAAADkmO9VAAAACXBIWXMAAAsTAAALEwEAmpwYAAACV0lEQVR42n2T208TQRSH928z4cWERH31QV+Um4VS8ALBaExLK/Wa+A/oiybEEEKgBZQE77FqChbBCA9ajHTbXdrupd2d6fw8My2ktIVJvuxcznyz5+yOdubsOXtl9Q3mFhJiPrmI+cUlLEiWlpFokJQsS14hQSRboHix+v4Dzl+4aGv9gUHIVmEcvsBRaN5rQo456mteEy7jyjE2cRPapZ5eIWV7BQP5/WKdYgn5fAH6zx/4nVnDr9Qn7G5uYG83i52VBHRdR6FYJkqKnGHC4zUMj4wK7XJvnzpVLuxbtqLkuLBsB5ZhoKznYOt7sPM5FOmw3M42rFIJJbdyGC/3SkcwNII2oVm2UPV9lUIN9VbxPLjVquqLxpznMxXbLBwcDnUWymAhBFzHAeccjA7wSSrnapwfKwwEh08Wli1626oU1d9MzkmOEw6FQienDMGbkjzaWoWM5kavXYfW09evBs1CnzH4XCC1Y+Dtlo7UdgHvtnJIpv/ideYfrdVUTMeUW4UGBTHOUK4w3H65gaGnaYw9XyfWEHyWxviL77BoTda2XRgkYf9AR6HcFJ3dJME6IjMZTBJSdms6A7vaWah+m96BK20pyxo6tGnu6y6mP/7B7JcsZlNZ1Z/5nIXrMaqh31bDkUYNa3RTBN0UQbekjlkUBXoyzxE13xXcdxSq77mCBCru4Cn3SsfVG+NcGwgE6l+NUuCNu8oatI6Po8oP7vIEtFNdXd8mY1PlO+GIQZiHRBoc9qNmOHbXjMbvmTEi/uCR+fDxEzM6FTfD0ZgRi9+3Tnd3p/8DFm0lEa5vFS8AAAAASUVORK5CYII=",
            },
            images: {
              fallback: {
                src: "/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png",
                srcSet:
                  "/static/bda88a04e99f6b9359a4b23f3b57f83a/48371/Group_5_Copy_3a01b21432.png 135w,\n/static/bda88a04e99f6b9359a4b23f3b57f83a/d56b7/Group_5_Copy_3a01b21432.png 270w,\n/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png 540w",
                sizes: "(min-width: 540px) 540px, 100vw",
              },
              sources: [],
            },
            width: 540,
            height: 417,
          },
        },
      },
    },
    DescriptionAlignment: "right",
    Description: {
      data: {
        childMarkdownRemark: {
          html: "<p>Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.</p>\n<p>Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.</p>",
        },
      },
    },
  },
];

describe("Test if the KeyInfoCard component is working as expected", () => {
  it("<KeyInfoCard /> matches snapshot", () => {
    const component = render(<KeyInfoCard data={cardData} />);
    expect(component.container).toMatchSnapshot();
  });

  it("<KeyInfoCard /> renders without exploding", () => {
    const component = render(<KeyInfoCard data={cardData} />);
    expect(component.getByTestId("key-info-card")).toBeInTheDocument();
  });
});
