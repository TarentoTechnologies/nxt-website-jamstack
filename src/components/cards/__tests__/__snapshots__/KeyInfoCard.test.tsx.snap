// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if the KeyInfoCard component is working as expected <KeyInfoCard /> matches snapshot 1`] = `
<div>
  <div
    class="container-fluid"
    data-testid="key-info-card"
  >
    <div
      class="px-3 px-sm-3 px-md-0 px-lg-0"
    >
      <div
        class="row  gx-5"
      >
        <div
          class="col-sm-12 col-md-6 col-lg-6 d-flex flex-column justify-content-center "
        >
          <div
            class="
      undefined
        
        
        infoSectionH2
        
        
        
        
        
        
        
        
        "
            data-aos="fade-up"
            data-testid="primary-title"
          >
            Challenge
          </div>
          <div
            class="py-2 black75"
            data-aos="fade-up"
          >
            <p>
              Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.
            </p>
            <p>
              Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.
            </p>
          </div>
        </div>
        <div
          class="col-sm-12 col-md-6 col-lg-6 "
        >
          <div
            class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
            data-gatsby-image-wrapper=""
            style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
          >
            <div
              style="max-width: 540px; display: block;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,%3Csvg%20height='400'%20width='540'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E"
                style="max-width: 100%; display: block; position: static;"
              />
            </div>
            <img
              alt=""
              aria-hidden="true"
              data-placeholder-image=""
              decoding="async"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAIAAABr+ngCAAAACXBIWXMAAAsTAAALEwEAmpwYAAADnklEQVR42gGTA2z8AL/F0N3g5dXY3ubn69TY3svR2MvQ2OLl6cPI0pSerqCotaKruKu0v7K5xLW7xr3Cy7/Ezb3Dy7q+yLm8xwDZ3eOZo7GnsL+xucWxuMOvt8KKlaXGy9K0vMiTna+ep7W3vsm5wMq5wMnT1t3U1dvM0NfN0djQ1NrV190A0tPXiJSiZ3OBdYGQj5yrwsjQoqq10tfd3+Dkp666k5yrr7K6qK23p665wsfPtLnDsrnEt7zHwMXO09XbANDQ04GJmFlDQlNQV5KZpMTEyZumtMTGzszDwLu4u09QVyspLImPmaSruLzCzIGEjJ6docDCyayyv7q6vwDAxMyCkaS8q621r7ijqLavsLaTnKmSm6mus7zm4+tnYGMAAARKUFmdpLGvtL9aXGW1rK61tbulsL7Av8YA09bdsKKi1MXDwrW4npSXsLS6jpWfgIeSlpmj/f//spiWYTw2lZeeho6aoKGngoWOy8rTmJKYt7zJzra2AN7k7caupdrNyKahpnRobsPIz6mvt5ieqJWXn+Xr+bqYkceilvj8/9XX38HEzKuttZqYnpWTlsGpnbOVhADuztuqi596cW52cniaoKrEw8empKasqrHTurbLp53Xt6j17+zu7/fw8vfe4efi5Ox/f4O/r53TwKbPxLUA1tTbtrjEm5OVlY2OkpWdnp6in56hm4R9oXlsx6iY++TV////9e/t4djc1tvl1NjgfH2DWFRUTkxPmpugAOTo8Obl6tLU2n1/hHBub9DFtu/fyu3dxuPUu8Ovm7CdltHT2sPCyIVzeIlrd8nDznZ5fz1CRlxCT49YaADm6vLq8PbJzNNnanK1q5/o1rzVxKzOu6Sij3x2a2Z3fIe5u8be4u+blqFpVFzIy9WBg4pGRkk4Mzuvm6kA5Onw6OzzzdLYYGNpVFVYd3V3cHBzTmN1RmN/DluTbKfS//z+8PL/y8rVioOKwMbQc3V7Pj9EKi4ytbe6AOLo7+Po7tbb41tfZzc6QHZ3flBpgQBZnQ5ytwBvuS6HxMjT5t7g68bJ1KestLm9xmpobFFRVUFES4iEigDh5u3g5uvb4OhobHQpKzBiYmZMYHYAOW8QLkwKToMAUZMlTnSJnLKtr7imq7StsbqanKObnqePlqKkp64A4+ju4+jt3+Pqf4KISElNY2RpbHF5K1mEM0tlNWOJIXCseoOKsrW7yc3XzM/ZztHaycvVvcHLusDKu8HKcgBLbWfKjJAAAAAASUVORK5CYII="
              style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
            />
            <img
              alt=""
              data-aos="fade-up"
              data-gatsby-image-ssr=""
              data-main-image=""
              data-src="/static/8db79876759e2148da5b7180b91a92d0/1535a/about_copy_bcd2468822.png"
              data-srcset="/static/8db79876759e2148da5b7180b91a92d0/c6d23/about_copy_bcd2468822.png 135w,/static/8db79876759e2148da5b7180b91a92d0/ba1a2/about_copy_bcd2468822.png 270w,/static/8db79876759e2148da5b7180b91a92d0/1535a/about_copy_bcd2468822.png 540w"
              decoding="async"
              loading="lazy"
              sizes="(min-width: 540px) 540px, 100vw"
              style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
            />
            <noscript />
            <script
              type="module"
            >
              const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
            </script>
          </div>
        </div>
      </div>
    </div>
    <div
      class="px-3 px-sm-3 px-md-0 px-lg-0"
    >
      <div
        class="row my-5 gx-5"
      >
        <div
          class="col-sm-12 col-md-6 col-lg-6 d-none d-md-block mt-4"
        >
          <div
            class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
            data-gatsby-image-wrapper=""
            style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
          >
            <div
              style="max-width: 540px; display: block;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,%3Csvg%20height='417'%20width='540'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E"
                style="max-width: 100%; display: block; position: static;"
              />
            </div>
            <img
              alt=""
              aria-hidden="true"
              data-placeholder-image=""
              decoding="async"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAYAAADkmO9VAAAACXBIWXMAAAsTAAALEwEAmpwYAAACV0lEQVR42n2T208TQRSH928z4cWERH31QV+Um4VS8ALBaExLK/Wa+A/oiybEEEKgBZQE77FqChbBCA9ajHTbXdrupd2d6fw8My2ktIVJvuxcznyz5+yOdubsOXtl9Q3mFhJiPrmI+cUlLEiWlpFokJQsS14hQSRboHix+v4Dzl+4aGv9gUHIVmEcvsBRaN5rQo456mteEy7jyjE2cRPapZ5eIWV7BQP5/WKdYgn5fAH6zx/4nVnDr9Qn7G5uYG83i52VBHRdR6FYJkqKnGHC4zUMj4wK7XJvnzpVLuxbtqLkuLBsB5ZhoKznYOt7sPM5FOmw3M42rFIJJbdyGC/3SkcwNII2oVm2UPV9lUIN9VbxPLjVquqLxpznMxXbLBwcDnUWymAhBFzHAeccjA7wSSrnapwfKwwEh08Wli1626oU1d9MzkmOEw6FQienDMGbkjzaWoWM5kavXYfW09evBs1CnzH4XCC1Y+Dtlo7UdgHvtnJIpv/ideYfrdVUTMeUW4UGBTHOUK4w3H65gaGnaYw9XyfWEHyWxviL77BoTda2XRgkYf9AR6HcFJ3dJME6IjMZTBJSdms6A7vaWah+m96BK20pyxo6tGnu6y6mP/7B7JcsZlNZ1Z/5nIXrMaqh31bDkUYNa3RTBN0UQbekjlkUBXoyzxE13xXcdxSq77mCBCru4Cn3SsfVG+NcGwgE6l+NUuCNu8oatI6Po8oP7vIEtFNdXd8mY1PlO+GIQZiHRBoc9qNmOHbXjMbvmTEi/uCR+fDxEzM6FTfD0ZgRi9+3Tnd3p/8DFm0lEa5vFS8AAAAASUVORK5CYII="
              style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
            />
            <img
              alt=""
              data-aos="fade-up"
              data-gatsby-image-ssr=""
              data-main-image=""
              data-src="/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png"
              data-srcset="/static/bda88a04e99f6b9359a4b23f3b57f83a/48371/Group_5_Copy_3a01b21432.png 135w,/static/bda88a04e99f6b9359a4b23f3b57f83a/d56b7/Group_5_Copy_3a01b21432.png 270w,/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png 540w"
              decoding="async"
              loading="lazy"
              sizes="(min-width: 540px) 540px, 100vw"
              style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
            />
            <noscript />
            <script
              type="module"
            >
              const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
            </script>
          </div>
        </div>
        <div
          class="col-sm-12 col-md-6 col-lg-6 d-flex flex-column justify-content-center mt-4"
        >
          <div
            class="
      undefined
        
        
        infoSectionH2
        
        
        
        
        
        
        
        
        "
            data-aos="fade-up"
            data-testid="primary-title"
          >
            Our Approach
          </div>
          <div
            class="py-2 black75"
            data-aos="fade-up"
          >
            <p>
              Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.
            </p>
            <p>
              Innovation is now more important than ever in today's world. To stand out in a crowded market, business leaders need to focus on different ways they can keep up with evolving demand. Innovation is now more important than ever in today's world.
            </p>
          </div>
        </div>
        <div
          class="col-sm-12 col-md-6 col-lg-6 d-block d-md-none mt-4"
        >
          <div
            class="gatsby-image-wrapper gatsby-image-wrapper-constrained"
            data-gatsby-image-wrapper=""
            style="position: relative; overflow: hidden; display: inline-block; vertical-align: top;"
          >
            <div
              style="max-width: 540px; display: block;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,%3Csvg%20height='417'%20width='540'%20xmlns='http://www.w3.org/2000/svg'%20version='1.1'%3E%3C/svg%3E"
                style="max-width: 100%; display: block; position: static;"
              />
            </div>
            <img
              alt=""
              aria-hidden="true"
              data-placeholder-image=""
              decoding="async"
              src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAYAAADkmO9VAAAACXBIWXMAAAsTAAALEwEAmpwYAAACV0lEQVR42n2T208TQRSH928z4cWERH31QV+Um4VS8ALBaExLK/Wa+A/oiybEEEKgBZQE77FqChbBCA9ajHTbXdrupd2d6fw8My2ktIVJvuxcznyz5+yOdubsOXtl9Q3mFhJiPrmI+cUlLEiWlpFokJQsS14hQSRboHix+v4Dzl+4aGv9gUHIVmEcvsBRaN5rQo456mteEy7jyjE2cRPapZ5eIWV7BQP5/WKdYgn5fAH6zx/4nVnDr9Qn7G5uYG83i52VBHRdR6FYJkqKnGHC4zUMj4wK7XJvnzpVLuxbtqLkuLBsB5ZhoKznYOt7sPM5FOmw3M42rFIJJbdyGC/3SkcwNII2oVm2UPV9lUIN9VbxPLjVquqLxpznMxXbLBwcDnUWymAhBFzHAeccjA7wSSrnapwfKwwEh08Wli1626oU1d9MzkmOEw6FQienDMGbkjzaWoWM5kavXYfW09evBs1CnzH4XCC1Y+Dtlo7UdgHvtnJIpv/ideYfrdVUTMeUW4UGBTHOUK4w3H65gaGnaYw9XyfWEHyWxviL77BoTda2XRgkYf9AR6HcFJ3dJME6IjMZTBJSdms6A7vaWah+m96BK20pyxo6tGnu6y6mP/7B7JcsZlNZ1Z/5nIXrMaqh31bDkUYNa3RTBN0UQbekjlkUBXoyzxE13xXcdxSq77mCBCru4Cn3SsfVG+NcGwgE6l+NUuCNu8oatI6Po8oP7vIEtFNdXd8mY1PlO+GIQZiHRBoc9qNmOHbXjMbvmTEi/uCR+fDxEzM6FTfD0ZgRi9+3Tnd3p/8DFm0lEa5vFS8AAAAASUVORK5CYII="
              style="height: 100%; left: 0px; position: absolute; top: 0px; width: 100%;"
            />
            <img
              alt=""
              data-aos="fade-up"
              data-gatsby-image-ssr=""
              data-main-image=""
              data-src="/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png"
              data-srcset="/static/bda88a04e99f6b9359a4b23f3b57f83a/48371/Group_5_Copy_3a01b21432.png 135w,/static/bda88a04e99f6b9359a4b23f3b57f83a/d56b7/Group_5_Copy_3a01b21432.png 270w,/static/bda88a04e99f6b9359a4b23f3b57f83a/0c77b/Group_5_Copy_3a01b21432.png 540w"
              decoding="async"
              loading="lazy"
              sizes="(min-width: 540px) 540px, 100vw"
              style="height: 100%; left: 0px; position: absolute; top: 0px; transform: translateZ(0); transition: opacity 250ms linear; width: 100%; will-change: opacity; opacity: 0;"
            />
            <noscript />
            <script
              type="module"
            >
              const t="undefined"!=typeof HTMLImageElement&&"loading"in HTMLImageElement.prototype;if(t){const t=document.querySelectorAll("img[data-main-image]");for(let e of t){e.dataset.src&&(e.setAttribute("src",e.dataset.src),e.removeAttribute("data-src")),e.dataset.srcset&&(e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset"));const t=e.parentNode.querySelectorAll("source[data-srcset]");for(let e of t)e.setAttribute("srcset",e.dataset.srcset),e.removeAttribute("data-srcset");e.complete&&(e.style.opacity=1,e.parentNode.parentNode.querySelector("[data-placeholder-image]").style.opacity=0)}}
            </script>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
